packages/lib-classifier/src/plugins/drawingTools/components/Ellipse/Ellipse.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import DragHandle from '../DragHandle'
+
+const GUIDE_DASH = [4, 4]
+const GUIDE_WIDTH = 1
+
+function Ellipse ({ active, children, mark, onFinish, scale, svg, tool }) {
```

Comment:
Should this be using the new SVG Context?

Reply:
Also `tool` isn't being used and can be removed.

Reply:
I think `svg` can go completely.